core/storage/classifier/gae_models.py

```diff
@@ -175,13 +183,35 @@ def create(
             interaction_id=interaction_id,
             exp_id=exp_id,
             exp_version=exp_version,
-            state_name=state_name, status=status, training_data=training_data
+            next_scheduled_check_time=next_scheduled_check_time,
+            state_name=state_name, status=status,
+            training_data=training_data,
             )
 
         training_job_instance.put()
         return instance_id
 
     @classmethod
+    def query_training_jobs(cls, cursor=None):
```

Comment:
@seanlip Aren't the tests in gae_models_test doing the same thing as what you are asking me to do?